llama-index-integrations/vector_stores/llama-index-vector-stores-upstash/tests/test_upstash.py

```diff
@@ -126,6 +125,7 @@ def test_upstash_vector_query(
     upstash_vector_store: UpstashVectorStore, text_nodes: List[TextNode]
 ) -> None:
     upstash_vector_store.add(nodes=text_nodes)
+    time.sleep(1)

```

Comment:
ooc: why did we need to add sleep here now?

Reply:
It's to make sure that upserted vectors are successfully indexed.